Improve docs - README

spier opened the pull request:
I made various fixes to the documentation.

Mostly fixed typos, some extra whitespace.
Also made some actual content changes like the git link and linking to the databricks website.

I am also leaving one comment in the review tab, where I think that the content likely points to an internal team at Shell that could be removed.

spier commented on README.md:
```diff
@@ -233,10 +226,13 @@ You are now ready to start developing your own functions. Please remember to fol
 
 # Support and contacts
+
 If you encounter any issues or have questions, please reach out to the Innersource-Enablement-Team. They will be happy to assist you and provide further information about the project.
```
I suspect the Innersource-Enablement-Team is an internal reference that should be removed?

Amber-Rigg replied:
We've fixed this in our latest pr to our main branch so people are directed to raise issues to reach out to us.

Amber-Rigg commented on README.md:
```diff
@@ -233,10 +226,13 @@ You are now ready to start developing your own functions. Please remember to fol
 > **_NOTE:_**  Ensure you have read the [Release Guidelines](RELEASE.md) before publishing your code.
```
We've updated our main branch now to include this to support open collaboration. The Release.md can now be found.

Amber-Rigg commented:
Thank you for your support with our read me, we've made some changes to our main branch. Please could you merge/rebase your branch with main and then update your pr to our develop branch so we can include your changes with our other changes to main.

spier commented:
@Amber-Rigg I rebased and fixed the merge conflicts. Then changed the base branch to develop.

Then there still was a conflict that was only shown in the GitHub UI, so I fixed that directly via the UI (don't actually understand why that happened).

It is probably easier to see what I changed when you step through the commits one by one.

In summary my changes are:

- link to databricks website
- change formatting of the logo
- spelling and whitespace fixes

Hope some of these might be useful.

Feel free to accept or reject as you see fit.

Amber-Rigg reviewed (approved):
Improved read me formatting to support the repo.
